### ISSUE TITLE
No speelchak

### ISSUE THREAD

**Issue by SomeKittens** [plaidchat/plaidchat#104]
Speelchak not workz in pliadkat.

(Linux Mint 17, Node 0.10.37)

**Comment by twolfson** [comment]
I don't understand this issue. Can you rephrase it?

**Comment by SomeKittens** [comment]
In the OSX Slack app, words I misspell are underlined, and I have the opportunity to right-click and correct them.  There's no such feature in plaidchat.

**Comment by twolfson** [comment]
Ah, we already have an issue for that, #89. We are waiting on the stable release of `nw.js@0.13` for support. Closing this issue as a duplicate.

**Comment by SomeKittens** [comment]
Good to hear it's coming down the pipe
